### PR TITLE
Rs 2023.01.26.retro

### PR DESCRIPTION
**Pull request by rschlaikjer**
2023-01-26-223754_1920x1140_scrot

### CODE REVIEW

File: script/Arcueid/Day 2/02_00_ARC02_1C.txt [hunk 1]
```diff
@@ -236,12 +236,12 @@ As I jog down the street, I don't spot anyone else wearing a school uniform.
 [sha:f1e3ddfff0d8f4a64386d180a7c4e491a1738749]{
 -- Page 108, Offset 2883.
```
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Arcueid/Day 2/02_00_ARC02_1C.txt:237]: 2868
>Yeah... I still can't...

File: script/Arcueid/Day 2/02_00_ARC02_1D.txt [hunk 2]
```diff
@@ -16,7 +16,9 @@ I use the remaining couple of minutes before homeroom starts to sit in my chair
 [sha:b78f105bf38252e25a7307b2a2831a396634bb53]{
 -- Page 114, Offset 2898.
 -- 公共による今朝の<報道|ニュース>。若い先生がたが率先して作った学校用の掲示板と、有志によって作られた生徒専用の匿名掲示板。
-This morning's news. The school bulletin board, created by a group of young teachers, and the anonymous student bulletin board, created by a group of volunteers.
+I read this morning's news.
+ I scan through the school bulletin board, created by a group of young teachers,
+ and the anonymous student bulletin board, created by a group of volunteers.
 }
 [sha:73ad7facad987db830f93f4a3199afdaa9f9f3d4]{
 -- Page 114, Offset 2899.
```
Comment (Petrikow) [inline script/Arcueid/Day 2/02_00_ARC02_1D.txt:24]: 2903
Misinterpreted
>It's just that dependence on the internet and dependence on friends in front of you have become equivalent. It's not like the people in our class don't get along.

File: script/Arcueid/Day 2/02_00_ARC02_1D.txt [hunk 3]
```diff
@@ -61,7 +63,8 @@ Thanks to how sophisticated the internet has become, the knowledge gap between g
 [sha:86febd0eba3130afe40aa8fa9d4f4ba0d0beaeec]{
 -- Page 117, Offset 2907.
 -- 確固として積み上げた物量の差はあるものの、生活していく上ではイーブンになったように見えてしまう。
-Although there is a difference in the number of things we have accumulated, we seem to have evened out in terms of lifestyles.
+Even though some people have accumulated more material goods than others,
+ we seem to have evened out in terms of lifestyles.
 }
 [sha:5bb0e0fcbd88c331effae5fa14f52a1c92aa39fd]{
 -- Page 118, Offset 2908.
```
Comment (Petrikow) [inline script/Arcueid/Day 2/02_00_ARC02_1D.txt:70]: 2917
>But hey, for how much I've been going on about this pretentious analysis, it's ultimately just my own thoughts.

Comment (Petrikow) [reply script/Arcueid/Day 2/02_00_ARC02_1D.txt:70]: Bunch of stuff in this fill needs to be consistent, and correct.
2909
>The world is small these days. Even elementary school students know how closed in our society has become.

2916
>I think that's what the depressing sense of being trapped in classrooms, schools, and cities is all about.

2918
>I don't feel like I'm trapped.

2919
>行き詰まりも倦怠感も、社会への不満もまるでない。
>Nor do I feel like I'm closed in, fatigued, or dissatisfied with society. Not at all.

File: script/Arcueid/Day 1/01_00_ARC01_8.txt [hunk 4]
```diff
@@ -431,7 +441,8 @@ Other than that... I heard that the family doctor of the previous head, Makihisa
 [sha:c62369a8cbe8609a120634b839a5ebe806e7931c]{
 -- Page 479, Offset 2375.
 -- 「三年もかぁ……なあ翡翠。そういうのって逗留っていうんじゃなくて、居候っていうんじゃないか？」
-"For three years? ...Hey, Hisui, weren't they more like freeloaders than guests then?"
+"Three whole years...?
+ Hey, Hisui, don't you think it might make more sense to call them 'freeloaders' instead of guests at that point?"
 }
 [sha:b54d179ba4d8d28d85e076ba4c6408a2e50d43fa]{
 -- Page 480, Offset 2376.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 1/01_00_ARC01_8.txt:448]: Not sure I buy it, I think this sentence is implicitly
> [As] the epitome of a servant, she doesn't say anything unnecessary.

which is all one connected thought.

**Pull request merged**